tools/detection2d/sample/yolox-s+-opt-T4-960x960.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
Please do not place standalone algorithm scripts such as `tools/detection2d/deploy_yolox_s_opt.py` or items like those in `tools/detection2d` under the tools directory.
If the tools only use for `YOLOX`, would you move into `projects/YOLOX_elan`?